cabal-hie-install

```diff
@@ -0,0 +1 @@
+cabal v2-run ./install.hs --project-file install/shake.project $@
```

Comment:
This should have `#!/bin/sh` on top, surely? and must be `chmod +x`'d too, if not already done.

Reply:
I already set the executable bit. I could execute as is in my msys bash but i'll add the sehbang just in case.